src/main/java/com/ft/content/model/ContentPackage.java

```diff
@@ -0,0 +1,52 @@
+package com.ft.content.model;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ContentPackage {
+
+    private final String description;
+
+    private final String listLink;
```

Comment:
you are right, it is a UUID. I will rename the field to `linkId`

I also need to do some changes in the mapper due to the fact that the actual structure of the link is different to what was initially expected.